--- origsrc/libvncserver-LibVNCServer-0.9.11/libvncclient/Makefile.am	2016-12-30 07:01:28.000000000 -0600
+++ src/libvncserver-LibVNCServer-0.9.11/libvncclient/Makefile.am	2017-02-14 01:34:06.537387300 -0600
@@ -25,5 +25,5 @@ EXTRA_DIST=corre.c hextile.c rre.c tight
 $(libvncclient_la_OBJECTS): ../rfb/rfbclient.h
 
 lib_LTLIBRARIES=libvncclient.la
-libvncclient_la_LDFLAGS = -version-info 1:0:0
+libvncclient_la_LDFLAGS = -version-info 1:0:0 -no-undefined
 
--- origsrc/libvncserver-LibVNCServer-0.9.11/libvncserver/Makefile.am	2016-12-30 07:01:28.000000000 -0600
+++ src/libvncserver-LibVNCServer-0.9.11/libvncserver/Makefile.am	2017-02-14 01:34:14.937435900 -0600
@@ -66,7 +66,7 @@ libvncserver_la_LIBADD += $(LIBSYSTEMD_L
 endif
 
 lib_LTLIBRARIES=libvncserver.la
-libvncserver_la_LDFLAGS = -version-info 1:0:0
+libvncserver_la_LDFLAGS = -version-info 1:0:0 -no-undefined
 
 if HAVE_RPM
 $(PACKAGE)-$(VERSION).tar.gz: dist
