--- origsrc/LibVNCServer-0.9.7/libvncclient/Makefile.am	2007-03-30 18:31:34.000000000 -0500
+++ src/LibVNCServer-0.9.7/libvncclient/Makefile.am	2009-02-19 13:30:52.305216500 -0600
@@ -2,6 +2,8 @@ AM_CFLAGS=-g -I $(top_srcdir) -I. -Wall
 
 libvncclient_la_SOURCES=cursor.c listen.c rfbproto.c sockets.c vncviewer.c minilzo.c
 
+libvncclient_la_LDFLAGS = -no-undefined
+
 noinst_HEADERS=lzoconf.h minilzo.h
 
 rfbproto.o: rfbproto.c corre.c hextile.c rre.c tight.c zlib.c zrle.c ultra.c
--- origsrc/LibVNCServer-0.9.7/libvncserver/Makefile.am	2008-01-31 20:30:35.000000000 -0600
+++ src/LibVNCServer-0.9.7/libvncserver/Makefile.am	2009-02-19 13:30:52.320841200 -0600
@@ -41,6 +41,8 @@ LIB_SRCS = main.c rfbserver.c rfbregion.
 
 libvncserver_la_SOURCES=$(LIB_SRCS)
 
+libvncserver_la_LDFLAGS = -no-undefined
+
 lib_LTLIBRARIES=libvncserver.la
 
 if HAVE_RPM
