x11vnc is shipped separately from libvncserver

--- origsrc/LibVNCServer-0.9.7/Makefile.am	2008-01-31 07:19:28.000000000 -0600
+++ src/LibVNCServer-0.9.7/Makefile.am	2009-02-19 13:30:51.992722500 -0600
@@ -4,7 +4,7 @@ if WITH_X11VNC
 X11VNC=x11vnc
 endif
 
-SUBDIRS=libvncserver examples contrib libvncclient vncterm classes client_examples test $(X11VNC)
+SUBDIRS=libvncserver examples contrib libvncclient vncterm classes client_examples test # $(X11VNC)
 DIST_SUBDIRS=libvncserver examples contrib libvncclient vncterm classes client_examples test
 
 bin_SCRIPTS = libvncserver-config
--- origsrc/LibVNCServer-0.9.7/client_examples/Makefile.am	2007-03-30 18:31:34.000000000 -0500
+++ src/LibVNCServer-0.9.7/client_examples/Makefile.am	2009-02-19 13:46:14.725005700 -0600
@@ -17,7 +17,7 @@ endif
 if HAVE_LIBSDL
 SDLVIEWER=SDLvncviewer
 
-SDLvncviewer_CFLAGS=$(SDL_CFLAGS)
+SDLvncviewer_CFLAGS=$(SDL_CFLAGS) $(AM_CFLAGS)
 
 # thanks to autoconf, this looks ugly
 SDLvncviewer_LDADD=$(LDADD) $(SDL_LIBS)
--- origsrc/LibVNCServer-0.9.7/configure.ac	2009-01-11 20:52:27.000000000 -0600
+++ src/LibVNCServer-0.9.7/configure.ac	2009-02-19 13:30:52.289591800 -0600
@@ -772,7 +772,8 @@ if test "$build_x11vnc" = "yes"; then
 	#
 	# configure.ac:690: required file `x11vnc/Makefile.in' not found
 	#
-	AC_CONFIG_FILES([x11vnc/Makefile x11vnc/misc/Makefile])
+	#AC_CONFIG_FILES([x11vnc/Makefile x11vnc/misc/Makefile])
+	:
 fi
 
 AC_CONFIG_COMMANDS([chmod-libvncserver-config],[chmod a+x libvncserver-config])
